File: UnitTestsBD/UnitTestsBD/Models/Tasks.cs
using Supabase.Postgrest.Attributes;
using Supabase.Postgrest.Models;


namespace UnitTestsBD.Models
{
    [Table("Tasks")]
    public class Tasks : BaseModel
    {
        [PrimaryKey("id")]
        public Guid Id { get; set; }

        [Column("id_goal")]
        public Guid IdGoal { get; set; }

        [Column("name_task")]
        public string NameTask { get; set; }

        [Column("description")]
        public string Description { get; set; }

        [Column("status")]
        public bool Status { get; set; }

        [Column("id_category")]
        public int IdCategory { get; set; }
    }
}

File: MakeYourselfApp/app/src/main/java/com/example/makeyourselfapp/models/database/Tasks.kt
package com.example.makeyourselfapp.models.database

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class Tasks(
    var id: String = "",
    @SerialName("id_goal")
    var idGoal: String = "",
    @SerialName("name_task")
    var nameTask: String? = "",
    var description: String? = "",
    var status: Boolean = false,
    @SerialName("id_category")
    var idCategory: Int = 1
)